.github/workflows/main.yml

```diff
@@ -326,7 +326,7 @@ jobs:
       - name: Install Python dependencies
         run: |
           python -m pip install -U pip 'setuptools>=47.0.0,<48'
-          python -m pip install scons pypiwin32 numpy ruamel.yaml cython h5py pandas pytest pytest-github-actions-annotate-failures
+          python -m pip install scons pypiwin32 numpy ruamel.yaml cython h5py pandas
```

Comment:
Can you please rebase onto the most recent `main` branch to remove these unrelated changes?